gsa/src/web/pages/reports/details/toolbaricons.js

```diff
@@ -146,7 +146,7 @@ const ToolBarIcons = ({
               showSuccessMessage={showSuccessMessage}
               showErrorMessage={showErrorMessage}
               showThresholdMessage={showThresholdMessage}
-              theshold={threshold}
+              threshold={threshold}
```

Comment:
Not sure if this requires a separate PR because it seems to me that this is also a bug fix for a functionality which might not have worked.